stdlib/xml/dom/minicompat.pyi

```diff
@@ -8,11 +9,13 @@ _T = TypeVar("_T")
 StringTypes: tuple[type[str]]
 
 class NodeList(list[_T]):
-    length: int
+    @property
+    def length(self) -> int: ...
     def item(self, index: int) -> _T | None: ...
 
 class EmptyNodeList(tuple[Any, ...]):
```

Comment:
What a bizarre class. If it's always meant to be, you know, empty, maybe this would be better?

```suggestion
class EmptyNodeList(tuple[()]):
```

Reply:
Why not :)